lms/static/js/spec/edxnotes/plugins/accessibility_spec.js

```diff
@@ -37,7 +37,9 @@ define([
         });
 
         afterEach(function () {
-            _.invoke(Annotator._instances, 'destroy');
+            while (Annotator._instances.length > 0) {
```

Comment:
We need to call destroy, which removes the element being destroyed from Annotator._instances.